mp/src/game/shared/momentum/mom_system_gamemode.h

```diff
@@ -30,9 +33,11 @@ class CGameModeBase : public IGameMode
     const char* GetStatusString() override { return "Playing"; }
     const char* GetDiscordIcon() override { return "mom"; }
     const char* GetMapPrefix() override { return ""; }
+    const char* GetGameModeCfg() override { return ""; }
```

Comment:
Should this be empty? Should we have a "default" config to execute as a base for all game modes? If so, you'll need to call CGameModeBase::ExecGameModeCfg() in every child class.

Reply:
I didn't think it necessary

Reply:
True, there is also `game.cfg` that gets executed on every map load as well